server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailGet.scala

```diff
@@ -77,7 +79,9 @@ case class EmailGetRequest(accountId: AccountId,
         htmlBody = bodyStructure.htmlBody,
         attachments = bodyStructure.attachments,
         headers = asEmailHeaders(mime4JMessage.getHeader),
-        bodyValues = bodyValues)
+        bodyValues = bodyValues,
+        hasAttachment = HasAttachment(!firstMessage.getLoadedAttachments.isEmpty),
```

Comment:
you can use nonEmpty if it is a scala collection

Reply:
It is not a scala collection. .asScala.nonEmpty seems overkill...